corehq/apps/userreports/reports/data_source.py

```diff
@@ -70,6 +71,10 @@ def get_data(self, slugs=None):
                         row[column_name] = '{:.0%}'.format(
                             float(row[column_name]) / total
                         )
+                elif report_column.format == 'month_name':
```

Comment:
yeah I think you should add a transform for this on the UI side as per below - though in the back of my head I think we will soon want to support transforms in data sources as well